src/insert/spacefinder/article-body-adverts.ts

```diff
@@ -288,14 +292,16 @@ const addDesktopInline2PlusAds = (): Promise<boolean> => {
 };
 
 const addMobileInlineAds = (): Promise<boolean> => {
+	const minDistanceFromArticleTop = isInAdDensityVariant ? 100 : 200;
```

Comment:
From testing on my real device (N=1), this ad is still below the fold by a fair way, so I think this should be OK.